django_async_orm/utils.py

```diff
@@ -12,9 +12,9 @@ def mixin_async_manager_factory(model):
     """
 
     base_manager_cls = model.objects.__class__
-    mixin_async_manager = type(f'MixinAsync{base_manager_cls.__name__}', (AsyncManager, base_manager_cls), dict())
-
-    return mixin_async_manager
+    if 'MixinAsyncUserManager' != f'MixinAsync{base_manager_cls.__name__}':
```

Comment:
is this happening because you have custom user model that inherited from `User` or `Abstract*User` ?

Reply:
Would it make more sense to test if the parent model have been already patched:
```python
if base_manager_cls.__name__.startswith('MixinAsync'):
  ...
```

Reply:
This should be in a different PR so we can merge it independently

Reply:
@rednaks 

maybe from doing `User = get_user_model()`

also : https://github.com/rednaks/django-async-orm/pull/10